Docs/Documentation/Configuration.md

```diff
@@ -159,14 +159,23 @@ To learn more about it please check the configurations for [Authentication](Auth
 
 ## Using the user's email to login
 
-You need to configure 2 things:
+You need to configure 2 things (version 9.0.4):
 
-* Change the Password identifier fields configuration to let it use the email instead of the username for user identify. Add this to your Application class, after CakeDC/Users Plugin is loaded.
+* Change the Password identifier fields and the Authenticator for Forms configuration to let it use the email instead of the username for user identify. Add this to your Application class, right before CakeDC/Users Plugin is loaded.
 
 ```php
-    $identifiers = Configure::read('Auth.Identifiers');
-    $identifiers['Password']['fields']['username'] = 'email';
-    Configure::write('Auth.Identifiers', $identifiers);
+        // Load more plugins here
+        $identifiers = Configure::read('Auth.Identifiers');
+        $identifiers['Password']['fields']['username'] = 'email';
+        Configure::write('Auth.Identifiers', $identifiers);
+
+        $authenticators = Configure::read('Auth.Authenticators');
+        $authenticators['Form']['fields']['username'] = 'email';
+        Configure::write('Auth.Authenticators', $authenticators);
+
+        //Configure::write('Users.config', ['users', 'permissions']);
```

Comment:
Is this line supposed to be commented-out here?

Reply:
I commented it out because not everyone likes to use custom Users.config files. Most probably don't. But earlier the documentation points to the config (to be set before load) so I tried to show the whole picture here.

Reply:
There's been some discussion about this file and we are actually working on it to allow more clear/flexible configuration overrides.